### ISSUE TITLE
detailed page: broken file links

### ISSUE THREAD

**Issue by StellaCh** [inspirehep/inspire-next#3159]
@drjova @Dinika can you please have a look into https://qa.inspirehep.net/literature/1266766 ?
![pdf_pic](https://user-images.githubusercontent.com/24955201/35726993-01de20c0-0807-11e8-989d-d879964eb9c3.PNG)

The links don't work and the name `PDF` is not clear enough for the users. Where are they supposed to redirect?

**Comment by salmele** [comment]
For future reference, have we done usability tests to understand the value of providing a vast amount of links to (the same representation of) the same thing? It is a long-standing issue in any case.

**Comment by Dinika** [comment]
I believe there could be some inconsistency in the backend when providing the links to the PDFs. 

When I add a record to literatures category through the 'Submit' form and approve it, the `documents` field added to the record is valid and directs to the right PDF. However, if I migrate the record directly to the literatures collections, the documents field is either not present at all, or it has incorrect link to the PDF (which results in an error or file not found page).

Also, I agree that perhaps in cases where multiple links are present for a record, we should display link to maybe just 1 pdf.

**Comment by StellaCh** [comment]
Requirements to be investigated with the users. I added a note in: https://docs.google.com/document/d/18ewBkOXoe6Rhvz0ciUIP-TtM5LzcVeznuB28Or15URM/edit#

**Comment by michamos** [comment]
indeed, as @Dinika is saying, this is a backend problem. We don't currently attach the files in the migration, but just put the links to the files in `documents`, as you can check by opening the record in the editor. All `documents` point to files on legacy, not to the URL on labs.
